src/Microsoft.DotNet.Build.Tasks/Targets/UpdateBuildNumber.targets

```diff
@@ -0,0 +1,43 @@
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="GetNextDailyBuildNumber"
+    AssemblyFile="$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll"
+    Condition="Exists('$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll')" />
```

Comment:
No, this is from a copy/paste.  I'll update this to be inline with the other targets files in this project that use the build tasks dll.